CodeRetreat/src/com/coderetreat/CodeRetreat.java

```diff
@@ -0,0 +1,54 @@
+package com.coderetreat;
+
+import java.util.Scanner;
+
+public class CodeRetreat {
```

Comment:
```suggestion
public class CodeRetreat {
```

Choose a better name for the class that would describe the intention of writing this class. For example, PrimeRangeNoLoop would have been an ideal name for this class